Docs 7456 fix terminology in policies categories topic pa (#213)
mulesoft/docs-api-manager: modules/ROOT/pages/available-policies.adoc

== Policy Categories

> Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the default policies by its category and the function it performs. 

Reviewer: ```suggestion
Policies are categorized by the function they perform. For example, because you can resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the default policies by each policy's category and the function it performs.
```

Author: This page is not needed. So removed it from the navigation.
